### DISCUSSION TITLE
I keep receiving the "You are unlikely to be verified until you have completed your GitHub user profile" and "The image you selected does not appear to contain your school name."

### DISCUSSION THREAD

**Discussion by programerunknown2580** [community/community#58500] [GitHub Education]
### Hi everyone,

i keep receiving the 2 messages in the title and i cant fix it i need help. i tried adding my name and a bio is there a certain amount of letters i should add to it?

**Comment by the-officialjosh** [comment]
These two issues can be addressed first in the place where you were you type in this your name `Patrick Andreasen` Now make sure this is how your name appears in your document and also you could try other documents like your transcripts, semester results...

**Comment by samus-aran** [comment]
Hi @programerunknown2580 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 I suspect this section would be most helpful.
